apps/wpcom-block-editor/src/wpcom/editor.scss

```diff
@@ -1 +1,7 @@
 @import "./features/use-classic-block-guide";
+
+.start-writing-hide {
+	.components-external-link {
+		display: none !important;
+	}
+}
```

Comment:
This hides "connect an account" in the Jetpack sidebar. In theory we could hide the Jetpack icon/button all together, but the icon displays/flashes before being hidden, so I thought I'd start by simply hiding the link that exits the flow. <img src="https://github.com/Automattic/wp-calypso/assets/140841/be7a9aee-d096-4eee-90e0-51508619ede0" width="200" />